### ISSUE TITLE
next and previous buttons code road local setup with screen readers.

### ISSUE THREAD

**Issue by blindgeekzone** [blindgeekzone/skills-introduction-to-github#7]
hi, using jaws for windows, ncda and windows narrator. totally blind. and so then set up the repo with docker desktop and then set it up in firefox, chrome, edge and brave. so then get the code road vs code extension. and then install that and using that. so then go to shift control p in vs code. then type coderoad:start and it starts. do the first lesson locally. then press control  and then the inde terminal comes up. then type in echo hello terminal. and then press enter. tells me the command succeeded. and then the prompt appears. now a couple of accessbiility issues. when i then go back to the code road interface and then hear get a hint which i do. then tells me capalization matters. and then press spacebar on run, nothing happens, just hear the link under neath. and then press the insert 8 for left mouse click, nothing. then press enter and then get the test runner filaed or nothing at all. so the screen reader is not being exposed to the controls for previous and next. can you please test this with jaws, ncda, windows narrator. and using firefox, chrome, edge and brave and all the same issues. so can you please fix htis so i then can do this indpenently. and have been waiting a couple of weeks to try to then trouble shoot with chat gtp help. and so cannot move forward in the work shop. also did contact the developer of the code road extension. never heard back. also contacted free code camp support. naomi kerigen community manager and quincy larson founder of free code camp, but no help. and in adelaide, australia. so the time zone difference. as 13.5 hours ahead of new york and 16.5 hours ahead of seattle. thanks.
